src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -557,7 +557,6 @@ void AuthenticationInAppListener::processErrorCode(int errorCode) {
       break;
 
     case 127:  // Invalid unblock code
-      aip->requestState(AuthenticationInApp::StateStart, this);
```

Comment:
Why this? At this point we are in some verification state. If not `start`, at least we need to be in a `unblock code needed`